pkg/antctl/command_definition.go

```diff
@@ -153,6 +154,12 @@ type flagInfo struct {
 	usage        string
 }
 
+type rawCommand struct {
```

Comment:
Probably add a few comments to explain what is a rawCommand.

Reply:
Done.